tests/protocol/mockwiredata.py

```diff
@@ -36,6 +38,7 @@
         "trans_prv": "wire/trans_prv",
         "trans_cert": "wire/trans_cert",
         "test_ext": "ext/sample_ext-1.3.0.zip",
+        "imds_info": "imds/valid.json",
```

Comment:
This file already existed, but a cursory search didn't show it being used in the codebase... There's also `imds/unicode.json` that seems to be identical.

Reply:
Huh, looks like it actually is in use; I had searched for `unicode.json`, but we're actually constructing the filename at runtime (i.e. `"{0}.json".format("unicode")` so I didn't see it at first.